apps/recovery-relay/lib/wallets/Jetton/index.ts

```diff
@@ -0,0 +1,186 @@
+import { Ton as BaseTon } from '@fireblocks/wallet-derivation';
+import { JettonMaster, TonClient, WalletContractV4 } from '@ton/ton';
+import { Address, beginCell, Cell, fromNano, toNano } from '@ton/core';
+import { AccountData } from '../types';
+import { defaultTonWalletV4R2code } from '../TON/tonParams';
+import axios from 'axios';
+import { LateInitConnectedWallet } from '../LateInitConnectedWallet';
+
+export class Jetton extends BaseTon implements LateInitConnectedWallet {
+  public memo: string | undefined;
+  public tokenAddress: string | undefined;
+  public decimals: number | undefined;
+
+  public setTokenAddress(address: string) {
+    this.tokenAddress = address;
+  }
+
+  public setDecimals(decimals: number) {
+    this.decimals = decimals;
+  }
+
+  public updateDataEndpoint(memo?: string): void {
+    this.memo = memo;
+  }
+
+  public getLateInitLabel(): string {
+    throw new Error('Method not implemented.');
+  }
```

Comment:
Do not leave this as throwing an error, please change it to some label. See ADA for an example